conf/modules/imu_common.xml

```diff
@@ -7,14 +7,24 @@
 
       This takes the IMU_GYRO_RAW, IMU_ACCEL_RAW and IMU_MAG_RAW ABI messages as input.
     </description>
+    <define name="IMU_INTEGRATION" value="FALSE" description="Enable gyro/accel integration calculations (enabled for the ekf2)"/>
+    <define name="IMU_GYRO_CALIB" value="{}" description="Gyroscope calibration structures list (see struct imu_gyro_t)"/>
+    <define name="IMU_ACCEL_CALIB" value="{}" description="Accelerometer calibration structures list (see struct imu_accel_t)"/>
+    <define name="IMU_MAG_CALIB" value="{}" description="Magnetometer calibration structures list (see struct imu_mag_t)"/>
+    <define name="IMU_GYRO_ABI_SEND_ID" value="ABI_BROADCAST" description="The gyro ABI ID which is send over telemetry/logging"/>
+    <define name="IMU_ACCEL_ABI_SEND_ID" value="ABI_BROADCAST" description="The accel ABI ID which is send over telemetry/logging"/>
+    <define name="IMU_MAG_ABI_SEND_ID" value="ABI_BROADCAST" description="The mag ABI ID which is send over telemetry/logging"/>
```

Comment:
I would prefer to have the structure
```
<section name="IMU" prefix="IMU_">
  <define name="INTEGRATION" ...
</section>
```
It tells the user to place the define outside the firmware section, which is much better in this section, especially for the calibration.

Reply:
Will change it